carbonmark/.generated/carbonmark-api-sdk/types/Def0.ts

```diff
@@ -0,0 +1,7 @@
+export type Def0 = {
```

Comment:
Can we give this `Def0` a more meaningful name such as `BlockchainNetwork` or just `Network` during code generation?